h/schemas/forms/admin/group.py

```diff
@@ -24,56 +22,65 @@
 VALID_GROUP_TYPES = (("restricted", _("Restricted")), ("open", _("Open")))
 
 
-@colander.deferred
-def group_creator_validator(node, kw):
-    def validate(form, value):
-        """
-        Validate that the creator username exists in the organization's authority.
+def username_validator(form, value):
```

Comment:
This function is used as a form-/schema-level validator (versus a single node validator). Does that warrant a comment or a rename?

Reply:
This reflects how it was named before. It was always a form level validator, but it's behavior was only to validate the creator. So it was called: `group_creator_validator`. Now it's doing all usernames I've broadened it to `username_validator`.

I think it's better to describe what it's trying to do, rather than how Colander forces us to do it.

Reply:
Added a comment now